Makefile

```diff
@@ -279,58 +284,54 @@ release-alias-tag: # Adds the tag to the last build tag.
 ## Development
 ## --------------------------------------
 
-.PHONY: create-cluster
-create-cluster: $(CLUSTERCTL) ## Create a development Kubernetes cluster on OpenStack using examples
-	$(CLUSTERCTL) \
-	create cluster -v 4 \
-	--bootstrap-flags="name=clusterapi" \
-	--bootstrap-type kind \
-	-m ./examples/_out/controlplane.yaml \
-	-c ./examples/_out/cluster.yaml \
-	-p ./examples/_out/provider-components.yaml \
-	-a ./examples/addons.yaml
-
-
 .PHONY: create-cluster-management
 create-cluster-management: $(CLUSTERCTL) ## Create a development Kubernetes cluster on OpenStack in a KIND management cluster.
-	kind create cluster --name=clusterapi
+	@if [ -z `kind get clusters | grep clusterapi` ]; then \
+		kind create cluster --name=clusterapi; \
+	fi
+	@if [ ! -z "${LOAD_IMAGE}" ]; then \
+		echo "loading ${LOAD_IMAGE} into kind cluster ..." && \
+		kind --name="clusterapi" load docker-image "${LOAD_IMAGE}"; \
+	fi
+	# Install cert manager.
+	kubectl \
+		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
+		apply -f https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
+	# Wait for cert-manager pods to be created
+	sleep 20
```

Comment:
anyway we wait below, why sleep here..

Reply:
If the pods have not been created the wait returns instantly